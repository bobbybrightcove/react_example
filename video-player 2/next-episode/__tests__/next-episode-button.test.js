import React from 'react';
import renderer from 'react-test-renderer';
import NextEpisodeButton from '../next-episode-button';


describe('NextEpisodeButton', () => {
  // assume current episode is on this url /watch/ef3b9004-9765-490b-b84e-38861ea1030a
  const player = {};
  const nextEpisodeID = '014e59c-ec94-4bba-aee8-bebcfe4283d9';

  it('renders according to valid snapshot', () => {
    const component = renderer.create(
      <NextEpisodeButton player={player} nextEpisodeID={nextEpisodeID} />
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
