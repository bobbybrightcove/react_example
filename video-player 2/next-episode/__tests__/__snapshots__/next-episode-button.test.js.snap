// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NextEpisodeButton renders according to valid snapshot 1`] = `
<a
  href="/watch/014e59c-ec94-4bba-aee8-bebcfe4283d9"
>
  <img
    alt="Next Episode"
    src="next-episode.svg"
  />
</a>
`;
